app/frontend/src/locales/en/translation.json

```diff
@@ -0,0 +1,148 @@
+{
+    "pageTitle": "AI Document Explorer",
+    "headerTitle": "AI Document Explorer",
+    "chat": "Chat",
+    "qa": "Ask a question",
+    "login": "Login",
+    "logout": "Logout",
+    "clearChat": "Clear chat",
+    "history": {
+        "chatHistory": "Chat history",
+        "openChatHistory": "Open chat history",
+        "noHistory": "No chat history",
+        "deleteModalTitle": "Delete chat history",
+        "deleteModalDescription": "This action cannot be undone. Delete this chat history?",
+        "deleteLabel": "Delete",
+        "cancelLabel": "Cancel",
+        "today": "Today",
+        "yesterday": "Yesterday",
+        "last7days": "Last 7 days",
+        "last30days": "Last 30 days"
+    },
+    "upload": {
+        "fileLabel": "Upload file:",
+        "uploadedFilesLabel": "Previously uploaded files:",
+        "noFilesUploaded": "No files uploaded yet",
+        "loading": "Loading...",
+        "manageFileUploads": "Manage file uploads",
+        "uploadingFiles": "Uploading files...",
+        "uploadedFileError": "Error uploading file - please try again or contact admin.",
+        "deleteFile": "Delete file",
+        "deletingFile": "Deleting file...",
+        "errorDeleting": "Error deleting.",
+        "fileDeleted": "File deleted"
+    },
+    "developerSettings": "Developer settings",
+
+    "chatEmptyStateTitle": "Chat with your data",
+    "chatEmptyStateSubtitle": "Ask anything or try an example",
+    "defaultExamples": {
```

Comment:
Is this config used anywere? We also have our own json config with example questions, so maybe good to write this down to avoid confusion?